packages/backend/src/api/resources/Domains.ts

```diff
@@ -0,0 +1,9 @@
+import type { DomainJSON } from './JSON';
+
+export class Domains {
```

Comment:
❓Shouldn't this extend BaseResource ?

Reply:
This whole class is probably not needed, since the `deleteDomain()` function will return a `DeletedObject`.

Reply:
@panteliselef BaseResource isn't present here like it is in clerkjs. The `factory` file handles the request

Reply:
My bad, didn't see the path of the file